packages/firebase/index.d.ts

```diff
@@ -8303,31 +8319,71 @@ declare namespace firebase.firestore {
     INTERNAL: { delete: () => Promise<void> };
   }
 
-  export interface LoadBundleTask {
+  /**
+   * Represents the task of loading a Firestore bundle. It provides progress of the bundle
+   * loading, task completion and error events should they be any.
+   *
+   * The API is compatible with `Promise<LoadBundleTaskProgress>`.
+   */
+  export interface LoadBundleTask extends PromiseLike<LoadBundleTaskProgress> {
+    /**
+     * Registers functions to listen to bundle loading progresses.
+     * @param next Called there is a progress update from bundle loading, typically whenever
+     *        a Firestore document is loaded it will generate a progress update.
```

Comment:
Please fix indent to match the rest of this file.

Reply:
Done.